index.d.ts

```diff
@@ -209,7 +209,7 @@ declare module 'react-table' {
     columns: HeaderColumn<D>[]
     state: State<D>
     debug?: boolean
-    loading: boolean
+    loading?: boolean
```

Comment:
yeah, this was just a mistake.  Loading is only required if you are dealing with server side data, it should always have been optional.

Reply:
:+1: ran into this working on #1545